less/home-page.less

```diff
@@ -1,9 +1,13 @@
+// *{
+//   border: 5px solid red;
+// }
+
+
 .home {
 
   .content-section {
     margin: 30px 0;
     display: flex;
-    flex-wrap: wrap;
     justify-content: space-between;
 
    
```

Comment:
Deleting up free lines (all the blank lines) will make your code look and feel more organized !